js/math.js

```diff
@@ -0,0 +1,12 @@
+//will eventually make into a class
+//multipurpose
+//will hold vectors, trig calcs, etc to reduce the same code w/in different objects;
+function calculateRotation(y, x, shift) {
```

Comment:
can just give shift the default value of 0 in the constructor and you wont need lines 5-10